code/controllers/subsystem/goldmansachs.dm

```diff
@@ -0,0 +1,137 @@
+SUBSYSTEM_DEF(goldmansachs)
+	name = "Economy"
+	wait = 5 MINUTES
+	init_order = INIT_ORDER_GOLDMANSACHS
+	flags = 0
+	var/roundstart_paychecks = 5
+	var/budget_pool = 35000
+	var/list/department_accounts = list(ACCOUNT_CIV = ACCOUNT_CIV_NAME,
+										ACCOUNT_ENG = ACCOUNT_ENG_NAME,
+										ACCOUNT_SCI = ACCOUNT_SCI_NAME,
+										ACCOUNT_MED = ACCOUNT_MED_NAME,
+										ACCOUNT_SRV = ACCOUNT_SRV_NAME,
+										ACCOUNT_CAR = ACCOUNT_CAR_NAME,
+										ACCOUNT_SEC = ACCOUNT_SEC_NAME)
+	var/list/generated_accounts = list()
+	var/full_ancap = FALSE // Enables extra money charges for things that normally would be free, such as sleepers/cryo/cloning.
```

Comment:
This should be a config option instead

Reply:
No. This isn't supposed to be enabled as a full thing, ever. It's bad game design and is exclusively an admin toy.

Reply:
i'll slap a secrets panel on it to account for VV not needing to exist